sdk/operationalinsights/Microsoft.Azure.OperationalInsights/src/Microsoft.Azure.OperationalInsights.csproj

```diff
@@ -0,0 +1,8 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>Microsoft Azure Operational Insights Data Library</Description>
+    <Description>Provides Microsoft Azure Operational Insights data operations</Description>
+    <AssemblyTitle>Microsoft Azure Operational Insights</AssemblyTitle>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+  </PropertyGroup>
```

Comment:
You changed the TargetFramework list from <SdkTargetFx>net452;net461;netstandard1.4;netstandard2.0</SdkTargetFx> to RequiredTargetFrameworks. We need to be sure we are including the correct set of target frameworks. Can you please file an issue tracking this? I suspect we will want to update RequiredTargetFrameworks to include these extra frameworks that SdkTargetFx includes.

Reply:
I have filed an issue for that https://github.com/Azure/azure-sdk-for-net/issues/5959